core/dropdowndiv.js

```diff
@@ -13,115 +13,139 @@
+const dom = goog.require('Blockly.utils.dom');
+const math = goog.require('Blockly.utils.math');
+const style = goog.require('Blockly.utils.style');
 
 
 /**
  * Class for drop-down div.
  * @constructor
  * @package
  */
-Blockly.DropDownDiv = function() {
-};
-
-/**
- * Drop-downs will appear within the bounds of this element if possible.
- * Set in Blockly.DropDownDiv.setBoundsElement.
- * @type {Element}
- * @private
- */
-Blockly.DropDownDiv.boundsElement_ = null;
-
-/**
- * The object currently using the drop-down.
- * @type {Object}
- * @private
- */
-Blockly.DropDownDiv.owner_ = null;
-
-/**
- * Whether the dropdown was positioned to a field or the source block.
- * @type {?boolean}
- * @private
- */
-Blockly.DropDownDiv.positionToField_ = null;
+const DropDownDiv = function() {};
 
 /**
  * Arrow size in px. Should match the value in CSS
  * (need to position pre-render).
  * @type {number}
  * @const
  */
-Blockly.DropDownDiv.ARROW_SIZE = 16;
+DropDownDiv.ARROW_SIZE = 16;
 
 /**
  * Drop-down border size in px. Should match the value in CSS (need to position
  * the arrow).
  * @type {number}
  * @const
  */
-Blockly.DropDownDiv.BORDER_SIZE = 1;
+DropDownDiv.BORDER_SIZE = 1;
 
 /**
  * Amount the arrow must be kept away from the edges of the main drop-down div,
  * in px.
  * @type {number}
  * @const
  */
-Blockly.DropDownDiv.ARROW_HORIZONTAL_PADDING = 12;
+DropDownDiv.ARROW_HORIZONTAL_PADDING = 12;
 
 /**
  * Amount drop-downs should be padded away from the source, in px.
  * @type {number}
  * @const
  */
-Blockly.DropDownDiv.PADDING_Y = 16;
+DropDownDiv.PADDING_Y = 16;
 
 /**
  * Length of animations in seconds.
  * @type {number}
  * @const
  */
-Blockly.DropDownDiv.ANIMATION_TIME = 0.25;
+DropDownDiv.ANIMATION_TIME = 0.25;
 
 /**
  * Timer for animation out, to be cleared if we need to immediately hide
  * without disrupting new shows.
  * @type {?number}
  * @private
  */
-Blockly.DropDownDiv.animateOutTimer_ = null;
+DropDownDiv.animateOutTimer_ = null;
 
 /**
  * Callback for when the drop-down is hidden.
  * @type {?Function}
  * @private
  */
-Blockly.DropDownDiv.onHide_ = null;
+DropDownDiv.onHide_ = null;
 
 /**
  * A class name representing the current owner's workspace renderer.
  * @type {string}
  * @private
  */
-Blockly.DropDownDiv.rendererClassName_ = '';
+DropDownDiv.rendererClassName_ = '';
 
 /**
  * A class name representing the current owner's workspace theme.
  * @type {string}
  * @private
  */
-Blockly.DropDownDiv.themeClassName_ = '';
+DropDownDiv.themeClassName_ = '';
+
+/**
+ * The content element.
+ * @type {!Element}
+ * @private
+ */
+DropDownDiv.DIV_;
+
+/**
+ * The content element.
+ * @type {!Element}
+ * @private
+ */
+DropDownDiv.content_;
+
+/**
+ * The arrow element.
+ * @type {!Element}
+ * @private
+ */
+DropDownDiv.arrow_;
+
+/**
+ * Drop-downs will appear within the bounds of this element if possible.
+ * Set in DropDownDiv.setBoundsElement.
+ * @type {Element}
```

Comment:
I think this should be {?Element} (and {?Object} below) in accordance with the new style guide.